modules/@apostrophecms/ui/ui/apos/components/AposTreeRows.vue

```diff
@@ -255,6 +256,11 @@ export default {
     });
   },
   methods: {
+    // Fix for chrome when some text is selected (needed double click to check the box)
+    // Comes from sortablejs, so we avoid the event to propagate to sortablejs listener
+    pointerEvent(event) {
+      event.stopPropagation();
+    },
     setHeights() {
       this.treeBranches.forEach(branch => {
         // Add padding to the max-height to avoid needing a `resize`
```

Comment:
This file changes are fixing the chrome double click on checkbox fix:
https://linear.app/apostrophecms/issue/PRO-7371/buggy-behavior-with-the-pages-manager-when-opening-it-from-a-link

Worksaround.